frontend/device/kobo/device.lua

```diff
@@ -722,6 +725,45 @@ end
 
 function Kobo:getUnexpectedWakeup() return unexpected_wakeup_count end
 
+local function writeToSys(val, file)
+    local f = io.open(file, "we")
+    if not f then
+        logger.info("Cannot open:", file)
+        return
+    end
+    local re, err_msg, err_code = f:write(val, "\n")
+    if not re then
+        logger.err("Error writing to value to file:", val, file, err_msg, err_code)
+    end
+    f:close()
+    return re
```

Comment:
Here it's clear due to context, but I'm still not a fan. It'd be better to rename it to something better like return_value for consistency. ;-)